### ISSUE TITLE
Poor color of scroll bars, undesirable in terms of contrast ratio.

### ISSUE THREAD

**Issue by nojiritakeshi** [microsoft/vscode#184233]
Does this issue occur when all extensions are disabled?: Yes

- VS Code Version: 1.78.2
- OS Version: Windows 10 22H2

**Steps to Reproduce:** 

1. Color Theme: dark modern (default)
2. Mouse over vertical scroll bar.

**What is expected?**

When the color scheme theme is Dark Modern, the scroll bars are light colored and difficult to use.
Although individuals can change the color scheme in settings.json, we believe the contrast ratio should be increased by default for accessibility reasons.

default:
![vscodeScrollbar](https://github.com/microsoft/vscode/assets/62943988/e565cb3e-b731-4ed2-909c-c65694837a42)
An example of a proposal for improvement:
![example](https://github.com/microsoft/vscode/assets/62943988/e69346a6-936b-420f-8018-489e84ff85dd)

**Comment by VSCodeTriageBot** [comment]
Thanks for creating this issue! It looks like you may be using an old version of VS Code, the latest stable release is 1.78.2. Please try upgrading to the latest version and checking whether this issue remains.

Happy Coding!
